uiplib/Wallpaper.py

```diff
@@ -0,0 +1,89 @@
+"""Module that gets and sets Wallpaper."""
+import sys
+
+
+class Wallpaper:
+    """Wallpaper Class that holds set and get functions."""
+
+    def set(self, filename):  # pragma: no cover
```

Comment:
why no cover? just have no cover on the part that deals with windows and macos right?